gp-res-filter/src/main/java/com/ibm/g11n/pipeline/resfilter/Bundle.java

```diff
@@ -26,20 +26,22 @@
     private Collection<ResourceString> resStrings;
 
     public Bundle () {
+        this.notes = null;
+        this.resStrings = null;
     }
 
     public Bundle(Collection<ResourceString> resStrings, List<String> notes) {
         if (resStrings != null) {
-            this.resStrings = new ArrayList<>(resStrings);
+            this.resStrings = new LinkedList<ResourceString>(resStrings);
         }
         if (notes != null) {
-            this.notes = new ArrayList<>(notes);
+            this.notes = new ArrayList<String>(notes);

```

Comment:
Let's use 'modern' style. Java 7+ does not require redundant type param. Because notes's type is `List<String>`, `<String>` is already known by compiler, and use of `<>` is the modern style.